tests/pages/jobs/JobJSONEditor-cy.js

```diff
@@ -0,0 +1,253 @@
+describe("Job JSON Editor", function() {
+  beforeEach(function() {
+    cy.configureCluster({
+      jobDetails: true,
+      mesos: "1-for-each-health",
+      nodeHealth: true
+    });
+    cy.visitUrl({ url: "/jobs/overview" });
+  });
+
+  it("renders proper JSON for a simple job", function() {
+    const jobName = "job-with-inline-shell-script";
+    const fullJobName = `${Cypress.env("TEST_UUID")}.${jobName}`;
+    const cmdline = "while true; do echo 'test' ; sleep 100 ; done";
+
+    // Click 'Create a job'
+    // Note: The current group contains the previous job
+    cy.get(".button.button-primary-link.button-narrow").click();
```

Comment:
Are you sure there is enough specificity here `.button.button-primary-link.button-narrow` haha